### PR TITLE
 update cache initialization to Promise.all and update yuml2svg package version and add test for readonly properties

### PR DESCRIPTION
**Pull request by amarzavery**
https://github.com/Azure/sway/pull/74

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "member-ordering": false,
     "no-unsafe-any": false,
     "no-null-keyword": false,
-    "prettier": true
+    "prettier": true,
+    "no-console": false
```
Comment (amarzavery) [inline tslint.json]: will remove this.

File: lib/validators/liveValidator.ts [hunk 2]
```diff
@@ -168,13 +168,11 @@ export class LiveValidator {
     // Construct array of swagger paths to be used for building a cache
     const swaggerPaths = await this.getSwaggerPaths()
     log.info(`Found ${swaggerPaths.length}`)
+    const promiseFactories = swaggerPaths.map(swaggerPath => {
+      return this.getSwaggerInitializer(swaggerPath)
```
Comment (sergey-shandar) [inline lib/validators/liveValidator.ts]: nit: `swaggerPath => this.getSwaggerInitializer(swaggerPath)`

**Comment by vladbarosan** [comment]
@billytrend  shouldnt the regression merge validation run also here ?

File: package.json [hunk 3]
```diff
@@ -47,7 +47,7 @@
     "@types/js-yaml": "^3.12.1",
     "@types/json-pointer": "^1.0.30",
     "@types/jsonpath": "^0.2.0",
-    "@types/lodash": "^4.14.123",
+    "@types/lodash": "4.14.123",
```
Comment (vladbarosan) [inline package.json]: can we create an issue for tracking these type problems

File: test/readOnlyValidationTests.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    const result = await validate.validateExamples(specPath, "Profiles_Update", {
+      consoleLogLevel: "off"
+    })
+    assert.strictEqual(
+      result.length,
+      2,
+      `swagger "${specPath}" with operation should contain 2 model validation errors.`
+    )
+    assert.strictEqual(
+      result[0].code === "READONLY_PROPERTY_NOT_ALLOWED_IN_REQUEST",
```
Comment (vladbarosan) [inline test/readOnlyValidationTests.ts]: >READONLY_PROPERTY_NOT_ALLOWED_IN_REQUEST [](start = 26, length = 40)

we need to add this to the errorConstants in validationError.ts

**Comment by billytrend** [comment]
I just pushed the fixes from my PR to this branch including the snapshot updates

**Pull request merged**